### ISSUE TITLE
【菁英计划】反向对齐过程中使用Adamax优化器对不齐

### ISSUE THREAD

**Issue by armijo1122** [PaddlePaddle/Paddle#36746]
版本、环境信息：
1)  PaddlePaddle-gpu==2.2.0rc0
2)  torch == 1.9.1
3)  Cuda == 10.2
4)  cudnn==7.6.5

8号文章《Unified Language Model Pre-training for Natural Language Understanding and Generation》，反向对齐时torch和paddle同时使用了内置的Adamax，第一次Loss在1e-5数量级，往后每一次反向传播后误差都会增加一个数量级

**Comment by tianxin1860** [comment]
> Loss

您好，请先确保网络结构、训练超参等是否完全对齐；前向是否已经对齐？

**Comment by armijo1122** [comment]
> > Loss
> 
> 您好，请先确保网络结构、训练超参等是否完全对齐；前向是否已经对齐？

您好，训练超参和前向已经对齐了，损失函数也已经对齐了，只是后续反向传播后无法对齐

**Comment by tianxin1860** [comment]
您好，已经收到反馈，Paddle Adamax 和 Pytorch Adamax 在实现上目前有 diff,  可以参见 #36712，我们会排期修复，请关注~
